Header header
float32[] prev_x
float32[] prev_y
float32[] cur_x
float32[] cur_y

